web/packages/teleterm/src/ui/services/resources/resourcesService.ts

```diff
@@ -124,11 +126,11 @@ export class ResourcesService {
         limit,
         search,
         query: '',
-        searchAsRoles: false,
+        searchAsRoles: includeRequestable,
         pinnedOnly: false,
         startKey: '',
         sortBy: { field: 'name', isDesc: true },
-        includeRequestable: false, //TODO(gzdunek): Support requestable resources in the search bar.
+        includeRequestable,
```

Comment:
Is there ever a situation where `searchAsRoles` and `includeRequestable` in the RPC won't have the same value?

Reply:
I believe we are going to be deprecating `searchAsRoles` in the future. `includeRequestable` handles all the cases we need (accessible, requestable, all) but I haven't added in the distinction yet. You might be able to get away with not adding `searchAsRoles` to this call at all now because of https://github.com/gravitational/teleport/blob/master/lib/web/apiserver.go#L2660

and
https://github.com/gravitational/teleport/blob/master/lib/auth/auth_with_roles.go#L1328

but i haven't looked at the tsh handler to confirm this

Reply:
Yeah, having these two options at the same time is confusing and we talked about deprecating it. 
Unfortunately, it may require adding another parameter instead of `includeRequestable` which is only a boolean. 
I think you Michael meant `includedResourceMode` (that has three values: accessible, requestable, all) which lives only in the web api layer.